extensions/twitch/src/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { ActionPanel, confirmAlert, Detail, getPreferenceValues, List, ListItem, OpenAction } from "@raycast/api";
+import React from "react";
+import fetch from 'node-fetch';
+
+import Item from "./interfaces/Item";
+import Preferences from "./interfaces/preferences";
+
+export default function main() {
+  const preferences: Preferences = getPreferenceValues();
+  const clientId = preferences.clientId;
+  const authorization = preferences.authorization;
+
+  const [loading, setLoading] = React.useState(false);
+  const [query, setQuery] = React.useState<string>("");
+  const [items, setItems] = React.useState<Item[]>([]);
+
+  React.useEffect(() => {
+    if (query.length == 0) return;
+    setLoading(true);
+
+    fetch(`https://api.twitch.tv/helix/search/channels?query=${query}&live_only=true`, {
+      headers: {
+        'Client-Id': clientId,
+        'Authorization': `Bearer ${authorization}`,
+      }
+    }).then(res => res.json()).then((data: any) => {
+      if (data && data.data) {
+        setItems(data.data);
+        setLoading(false);
+      } else if (data.error && data.error.toLowerCase().includes("invalid")) {
+        confirmAlert({
+          title: "Error",
+          message: data.message,
+        });
+      }
+    });
+  }, [query]);
+
+  return (<>
+    <List isLoading={loading} searchBarPlaceholder="Search for a Streamer on Twitch" navigationTitle="Search a Channel" onSearchTextChange={(text) => setQuery(text)}>
+      {items.map((item: Item) => {
+        return <ListItem key={item.id} icon={item.thumbnail_url} id={item.id} title={item.title} subtitle={`${item.display_name} - ${item.is_live ? `${item.game_name}`:"Offline"}`} actions={
```

Comment:
@dieserRobin,
a suggestion here to make your extension even better.

Instead of set all that information together, would be better for the UX of your extension if you make use of the `accessoryTitle` and `accessoryIcon`.

![CleanShot 2022-01-03 at 22 34 35](https://user-images.githubusercontent.com/83694/147982843-bd2a593a-cd5c-47ee-99ef-afd83e050504.png)

You can set in the `accessoryIcon` two icons with different colors:
- For online: 
  - Icon: `Checkmark`
  - Color: `Green`
- For offline: 
  - Icon: `XmarkCircle`
  - Color: `Red`

You can the `accessoryTitle` to set the `game_name` in case the channel is broadcasting some game and set 'Offline' in case of not active.

Maybe another interesting thing for you to do is to mask the thumbnail with a circle instead of keeping it squared, this may bring a nice look and feel.

Reply:
@Aayush9029 Thanks for your positive review, I legit did this earlier only on a half hour because I was missing the feature in Raycast. I have now implemented your two suggestions.

@unnamedd, Thank you also for your suggestions for improvement, I have now implemented them too.